@xen-orchestra/vmware-explorer/esxi.mjs

```diff
@@ -294,12 +294,13 @@ export default class Esxi extends EventEmitter {
           if (typeof disk !== 'object') {
             continue
           }
-          // can be something other than a disk, like a controller card
-          if (channelType === 'scsi' && disk.deviceType !== 'scsi-hardDisk') {
+          //
+          if (disk.deviceType.match(/cdrom/i)) {
             continue
           }
+          // can be something other than a disk, like a controller card
           // ide hard disk don't have deviceType, but cdroms have one
```

Comment:
Isn't that commentary related to the line `if (disk.deviceType.match(/cdrom/i))` above ? If it doesn't, we can remove the empty commentary above that line.